packages/strapi-hook-bookshelf/lib/buildDatabaseSchema.js

```diff
@@ -383,6 +383,7 @@ const getType = ({ definition, attribute, name, tableExists = false }) => {
       return client === 'pg' ? 'double precision' : 'double';
     case 'decimal':
       return 'decimal(10,2)';
+    // TODO: split time types as they should be different
```

Comment:
Also timestamp vs datetime as timestamp does do some conversions hence why it is normally only used for fields like `created_at` and `updated_at`:

MySQL converts TIMESTAMP values from the current time zone to UTC for storage, and back from UTC to the current time zone for retrieval. (This does not occur for other types such as DATETIME.)

https://dev.mysql.com/doc/refman/8.0/en/datetime.html

Reply:
Yes the next step will be to manage each of the types separatly. right now I'm aiming at being able to send and receive UTC iso dates so it is consistent ;)

Reply:
Perfect, that's what I thought I was going to look at splitting them also when develop get's merged. But I'll test this branch to make sure this patch is working